### PR TITLE
[BUG] Fix sending blank messages in reply and quote message.

### LINKED ISSUE #2070: [BUG] Should not send spaces in reply and quote message.

### PR DESCRIPTION
**Pull request by Cool-fire**
@RocketChat/android

Closes #2070

### CODE REVIEW

File: app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt [hunk 1]
```diff
@@ -868,8 +868,10 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
 
             button_send.setOnClickListener {
                 var textMessage = citation ?: ""
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt]: No need to have `var textMessage = citation ?: ""`
`if (!citation.isNullOrBlank()) { ... }` is the way to go.

**Review by philipbrito — CHANGES_REQUESTED** [review]

File: app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt (hunk 1 above)
Comment (Cool-fire) [reply app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt]: Thanks @filipedelimabrito for comments. I think it we should keep `var textMessage = citation ?: ""` we are adding `text_message.textContent` to textMessage. And textMessage holds **citation** if it is there else a empty string. For the bug I just added `if(!text_message.textContent.isBlank())` to check if the input message is blank and the line `var textMessage = citation ?: ""` exists from previous codebase .

Comment (Cool-fire) [reply app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt]: Let me know if I have to change the code. I will do it and update the pull request.

**Review by philipbrito — APPROVED** [review]
I've made some changes here.
Thanks for sending this PR @Cool-fire

**Pull request merged**